README.md

```diff
@@ -67,6 +67,14 @@ emerge x11-terms/alacritty
 pkg install alacritty
 ```
 
+### macOS
+
+```sh
+brew cask install alacritty
+```
+
+Note: See the relevant installation instructions to install Alacritty's [manual page](INSTALL.md#manual-page), [shell completions](INSTALL.md#shell-completions), and [Terminfo](INSTALL.md#terminfo).
```

Comment:
Rather than using `Note:` as a kind of annotation, I think a proper sentence would be nicer to read here.

Something like this:

Once the cask is installed, it's recommended to setup the [manual page](), [shell completions]() and [terminfo definitions]().

Better suggestions are always welcome of course.

Reply:
Looks good to me!